README.md

```diff
@@ -194,6 +194,74 @@ for getting the file list.
 This option cannot be set to `false` with `redirect` set to `true` on a server
 with `ignoreTrailingSlash` set to `true`.
 
+#### `hash`
+
+Default: `undefined`
+
+To enable `hash`, the `wildcard` option must be turned off. When enabled, `hash` lets the user access assets dynamically using the decorated `getHashedAsset` function. This in turn makes possible the usage of a very high `maxAge` so that the content can be cached as long as possible. If any modifications are made to a file, its hash will simply be recalculated during the next startup and the cache will bust for that asset.
```

Comment:
```suggestion
To enable `hash`, the `wildcard` option must be turned off.
```
I think it should be mention that when `hash` is true, `wildcard` is false by default

Reply:
Done